src/stylesheets/components/_alerts.scss

```diff
@@ -88,14 +88,14 @@ $icon-size: 3rem;
 }
```

Comment:
Use `@extend %usa-heading;` in `.usa-alert-heading` to explicitly get the heading styles

Reply:
Is that in lieu of calling margin-top/bottom? Like we want to add all the additional styles (like `line-height`) to this header as well?

Reply:
yup — it needs font size/face/line-height etc now that these properties aren't being applied via the `h3` element itself

Reply:
Also, i made this change :)